lib/Doctrine/Common/Collections/ExpressionBuilder.php

```diff
@@ -163,4 +163,15 @@ public function contains($field, $value)
     {
         return new Comparison($field, Comparison::CONTAINS, new Value($value));
     }
+
+    /**
+     * @param string $field
+     * @param mixed  $value
+     *
+     * @return Comparison
+     */
+    public function memberOf($value, $field)
```

Comment:
Can you please change the parameter order for consistency with all the other methods ?